pkg/init/asker/asker.go

```diff
@@ -77,3 +78,121 @@ func (o *Survey) AskName(defaultName string) (string, error) {
 	}
 	return answer, nil
 }
+
+// AskPersonalizeConfiguration asks the configuration user wants to change
+func (o *Survey) AskPersonalizeConfiguration(configuration ContainerConfiguration) (ContainerMap, error) {
+	options := []string{
```

Comment:
If you want to run unit test on building the options/tracker, it would be helpful to extract the first part of this method to a specific method that builds options/tracker based on `configuration`, so you can test the options/tracker content based on different ports/envs.

Reply:
So, writing a method proved to be a pain for me while writing unit test for `PersonalizeDevfileconfig` method of `interactive.go`. Since it's purely logical/mathematical, I decided to write a function, and testing shouldn't be difficult.